lms/djangoapps/bulk_email/models.py

```diff
@@ -83,32 +165,41 @@ def __unicode__(self):
 
     @classmethod
     def create(
-            cls, course_id, sender, to_option, subject, html_message,
-            text_message=None, template_name=None, from_addr=None):
+            cls, course_id, sender, targets, subject, html_message,
+            text_message=None, template_name=None, from_addr=None, cohort_name=None):
         """
         Create an instance of CourseEmail.
         """
         # automatically generate the stripped version of the text from the HTML markup:
         if text_message is None:
             text_message = html_to_text(html_message)
 
-        # perform some validation here:
-        if to_option not in TO_OPTIONS:
-            fmt = 'Course email being sent to unrecognized to_option: "{to_option}" for "{course}", subject "{subject}"'
-            msg = fmt.format(to_option=to_option, course=course_id, subject=subject)
-            raise ValueError(msg)
+        new_targets = []
+        for target in targets:
+            # Ensure our desired target exists
+            if target not in EMAIL_TARGETS:
+                fmt = 'Course email being sent to unrecognized target: "{target}" for "{course}", subject "{subject}"'
+                msg = fmt.format(target=target, course=course_id, subject=subject)
+                raise ValueError(msg)
```

Comment:
Depends on what you mean by "will". These are backend API-level errors, so our frontend should not be hitting these cases. They're not really intended to be surfaced to the end user; it's more likely they'd be used when an operator is digging through log files to diagnose an unexpected error. Do we traditionally translate such strings?